pennylane_lightning/src/gates/cpu_kernels/avx_common/ApplyCNOT.hpp

```diff
@@ -69,13 +67,14 @@ template <typename PrecisionT, size_t packed_size> struct ApplyCNOT {
     }
 
     template <size_t control> static constexpr auto maskInternalExternal() {
-        std::array<bool, packed_size> mask = {
-            false,
-        };
+        std::array<bool, packed_size> mask{};
```

Comment:
Nope, it was changed as I found that `={false,}` only initializes the first element. As it may confuse readers, I just choose to do not to write it explicitly.